src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,67 @@
+package calculator;
+
+import calculator.utils.validator.InputValidator;
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
+
+public class StringAddCalculator {
+
+    private static final String DEFAULT_DELIMETER_REGEX = ",|:";
+    private static final String CUSTOM_DELIMETER_REGEX = "//(.)\n(.*)";
+
+    private static final int CUSTOM_DELIMETER = 1;
+    private static final int CUSTOM_PATTERN_NUMBERS = 2;
+
+
+    private static final Pattern customDelimeterPattern = Pattern.compile(CUSTOM_DELIMETER_REGEX);
```

Comment:
CUSTOM_DELIMETER_REGEX 이 별도로 사용될일이 없다면, 굳이 상수로 선언되는 Pattern의 매개변수도 상수로만들진 않아도 될 것 같네요!
```suggestion
    private static final Pattern customDelimeterPattern = Pattern.compile("//(.)\n(.*)");
```